src/ai/susi/mind/SusiSkill.java

```diff
@@ -617,6 +619,28 @@ public static JSONObject getSkillRating(String model, String group, String langu
         return newRating;
     }
 
+    public static boolean getSkillStatus(String model, String group, String language, String skillname) {
+        // skill status
+        JsonTray skillStatus = DAO.skillStatus;
+        if (skillStatus.has(model)) {
+            JSONObject modelName = skillStatus.getJSONObject(model);
+            if (modelName.has(group)) {
+                JSONObject groupName = modelName.getJSONObject(group);
+                if (groupName.has(language)) {
+                    JSONObject languageName = groupName.getJSONObject(language);
+                    if (languageName.has(skillname)) {
+                        JSONObject skillName = languageName.getJSONObject(skillname);
+
+                        if (skillName.has("reviewed")) {
```

Comment:
It checks if the entry is present or not. Suppose, if a admin wants to change the status to false back again. In that case, it will fail. Read the value of `reviewed` and then return that value.

Reply:
If the admin wants to change the status of any Skill back to `false` again, then he/she can remove the entry of the Skill from the `skillStatus.json` file.

Reply:
Yes, this can be done.
But make sure to incorporate it in the API implementation.
Rest looks good 👍

Reply:
Yes, surely. Thanks :-)